packages/core/contracts/oracle/test/MockOracleAncillary.sol

```diff
@@ -44,6 +44,8 @@ contract MockOracleAncillary is OracleAncillaryInterface, Testable {
     mapping(bytes32 => mapping(uint256 => mapping(bytes => QueryIndex))) private queryIndices;
     QueryPoint[] private requestedPrices;
 
+    event PriceRequestAdded(uint256 indexed roundId, bytes32 indexed identifier, uint256 time);
```

Comment:
This event is added so that we can more easily test that the Handler is requesting a price correctly from this contract.